portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,36 +19,55 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+
 import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+import java.util.ArrayList;
 import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
 import com.andrewrs.sps.data.ListRecord;
-import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     private ArrayList<ListRecord> records;
-  public void init()
-  {
-    records = new ArrayList<ListRecord>();    
-  }
+  public void init(){}
```

Comment:
Nit: Defining `init()` as an empty method is not needed in this case so you can delete this line altogether.

Reply:
dine in next pr(7)